specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2025-07-01-preview/dataConnectors.json

```diff
@@ -1180,6 +1190,69 @@
       ],
       "type": "object"
     },
+    "PremiumMicrosoftDefenderForThreatIntelligence": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataConnector"
+        }
+      ],
+      "description": "Represents Premium Microsoft Defender for Threat Intelligence data connector.",
```

Comment:
>Premium Microsoft Defender for Threat Intelligence [](http://example.com/codeflow?start=33&length=50)

Curious should this be described as "Microsoft Defender Threat Intelligence Premium data connector."

Reading through [https://techcommunity.microsoft.com/blog/defenderthreatintelligence/introducing-the-mdti-premium-data-connector-for-sentinel/4220346](https://techcommunity.microsoft.com/blog/defenderthreatintelligence/introducing-the-mdti-premium-data-connector-for-sentinel/4220346) and [https://www.microsoft.com/en-us/security/business/siem-and-xdr/microsoft-defender-threat-intelligence](https://www.microsoft.com/en-us/security/business/siem-and-xdr/microsoft-defender-threat-intelligence)